content/efcore/getting-started/samples/configuring-deletion/manual-deletion-strategy/Program.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var reader = context.Readers.Include(r => r.Address).First();
+            context.Readers.Remove(reader);
+            context.SaveChanges();
+
+            var readers = context.Readers.Include(r => r.Address).ToList();
+            foreach (Reader rd in readers)
```

Comment:
Instead of printing out every reader, let's just print out the reader we updated before and after calling `SaveChanges` so that they can see the change.